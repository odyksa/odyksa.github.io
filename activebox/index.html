<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="utf-8">
		<title>ActiveBox</title>
		<!-- main css styles -->
		<link rel="stylesheet" href="styles/css/styles.css">
		<!-- script for IE6,7,8 to support html5 elements -->
		<!--[if lt IE 9]>
  			<script src="scripts/html5shiv/html5shiv.js"></script>	
  		<![endif]-->			
	</head>
	<body>
		<div class="page">

			<!-- start -->
			<section class="start" style="background-image: url(images/start_bg.jpg);">
				<div class="container">	
					<div class="start__inner">				
						
						<!-- header -->
						<header class="start__header">

								<!-- logo -->
								<div class="logo">
									<a href="index.html" class="logo__link" style="background-image: url(images/logo.png);"></a>
								</div>
								<!-- /logo -->

								<!-- nav -->
								<nav class="nav">
									<ul class="nav__list">
										<li class="nav__item"><a href="#features" class="nav__link">Features</a></li>
										<li class="nav__item"><a href="#works" class="nav__link">Works</a></li>
										<li class="nav__item"><a href="#team" class="nav__link">our team</a></li>
										<li class="nav__item"><a href="#clients" class="nav__link">Testimonials</a></li>
										<li class="nav__item"><a href="#download" class="nav__link">Download</a></li>
									</ul>
								</nav>
								<!-- /nav -->
							
						</header>
						<!-- /header -->

						<!-- content -->
						<section class="start__content">
							<h1 class="start__title">Your favorite one page multi purpose template</h1>
							<p class="start__descr">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent commodo cursus magna vel scelerisque nisl consectetur et.</p>
							<button class="button">Find out more</button>
						</section>
						<!-- /content -->

					</div>
				</div>
			</section>
			<!-- /start -->

			<!-- features -->
			<section class="features">
				<a id="features"></a>
				<div class="container">
					<div class="features__items">
						<div class="features__item">
							<div class="features__icon" style="background-image: url(images/features-item-1.png);">&nbsp;</div>
							<h2 class="features__title">Easily Customised</h2>
							<p class="features__descr">Cras justo odio, dapibus ac facilisis in, egestas eget quam. Nullam quis risus eget urna mollis ornare vel eu leo. Donec ullamcorper nulla non metus auctor.</p>
						</div>
						<div class="features__item">
							<div class="features__icon" style="background-image: url(images/features-item-2.png);">&nbsp;</div>
							<h2 class="features__title">Easily Customised</h2>
							<p class="features__descr">Cras justo odio, dapibus ac facilisis in, egestas eget quam. Nullam quis risus eget urna mollis ornare vel eu leo. Donec ullamcorper nulla non metus auctor.</p>
						</div>
						<div class="features__item">
							<div class="features__icon" style="background-image: url(images/features-item-3.png);">&nbsp;</div>
							<h2 class="features__title">Easily Customised</h2>
							<p class="features__descr">Cras justo odio, dapibus ac facilisis in, egestas eget quam. Nullam quis risus eget urna mollis ornare vel eu leo. Donec ullamcorper nulla non metus auctor.</p>
						</div>
						<div class="features__item">
							<div class="features__icon" style="background-image: url(images/features-item-4.png);">&nbsp;</div>
							<h2 class="features__title">Easily Customised</h2>
							<p class="features__descr">Cras justo odio, dapibus ac facilisis in, egestas eget quam. Nullam quis risus eget urna mollis ornare vel eu leo. Donec ullamcorper nulla non metus auctor.</p>
						</div>
						<div class="features__item">
							<div class="features__icon" style="background-image: url(images/features-item-5.png);">&nbsp;</div>
							<h2 class="features__title">Easily Customised</h2>
							<p class="features__descr">Cras justo odio, dapibus ac facilisis in, egestas eget quam. Nullam quis risus eget urna mollis ornare vel eu leo. Donec ullamcorper nulla non metus auctor.</p>
						</div>
						<div class="features__item">
							<div class="features__icon" style="background-image: url(images/features-item-6.png);">&nbsp;</div>
							<h2 class="features__title">Easily Customised</h2>
							<p class="features__descr">Cras justo odio, dapibus ac facilisis in, egestas eget quam. Nullam quis risus eget urna mollis ornare vel eu leo. Donec ullamcorper nulla non metus auctor.</p>
						</div>
					</div>
				</div>
			</section>
			<!-- /features -->

			<!-- works -->
			<section class="works">
				<a id="works"></a>
				<div class="works__item">
					<a href="#" class="works__link" style="background-image: url(images/work1.png);">&nbsp;</a>
					<div class="works__summary hidden">
						<h2 class="works__title">Project name</h2>
						<p class="works__descr">User interface design</p>					
					</div>					
				</div>
				<div class="works__item">
					<a href="#" class="works__link" style="background-image: url(images/work2.png);">&nbsp;</a>
					<div class="works__summary hidden">
						<h2 class="works__title">Project name</h2>
						<p class="works__descr">User interface design</p>					
					</div>					
				</div>
				<div class="works__item">
					<a href="#" class="works__link" style="background-image: url(images/work3.png);">&nbsp;</a>
					<div class="works__summary hidden">
						<h2 class="works__title">Project name</h2>
						<p class="works__descr">User interface design</p>					
					</div>					
				</div>
				<div class="works__item">
					<a href="#" class="works__link" style="background-image: url(images/work4.png);">&nbsp;</a>
					<div class="works__summary hidden">
						<h2 class="works__title">Project name</h2>
						<p class="works__descr">User interface design</p>					
					</div>					
				</div>
				<div class="works__item">
					<a href="#" class="works__link" style="background-image: url(images/work5.png);">&nbsp;</a>
					<div class="works__summary hidden">
						<h2 class="works__title">Project name</h2>
						<p class="works__descr">User interface design</p>					
					</div>					
				</div>
				<div class="works__item">
					<a href="#" class="works__link" style="background-image: url(images/work6.png);">&nbsp;</a>
					<div class="works__summary hidden">
						<h2 class="works__title">Project name</h2>
						<p class="works__descr">User interface design</p>					
					</div>					
				</div>
				<div class="works__item">
					<a href="#" class="works__link" style="background-image: url(images/work7.png);">&nbsp;</a>
					<div class="works__summary hidden">
						<h2 class="works__title">Project name</h2>
						<p class="works__descr">User interface design</p>					
					</div>					
				</div>
				<div class="works__item">
					<a href="#" class="works__link" style="background-image: url(images/work8.png);">&nbsp;</a>
					<div class="works__summary hidden">
						<h2 class="works__title">Project name</h2>
						<p class="works__descr">User interface design</p>					
					</div>					
				</div>				
			</section>
			<!-- /works -->

			<!-- team -->
			<section class="team">
				<a id="team"></a>
				<div class="container">
					<div class="team__items">
						<div class="team__item">
							<div class="team__pic" style="background-image: url(images/team-item-1.png);"></div>
							<h2 class="team__name">Ruth Woods</h2>
							<span class="team__function">Founder, CEO</span>
							<p class="team__descr">Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Maecenas sed diam eget risus varius blandit sit amet non magna. Nullam quis risus eget urna mollis ornare vel eu leo.</p>
							<div class="team__contacts social">
								<a href="#" class="social__link social__link_fb" style="background-image: url(images/social-icons.png);">&nbsp;</a><a href="#" class="social__link social__link_tw" style="background-image: url(images/social-icons.png);">&nbsp;</a><a href="#" class="social__link social__link_ln" style="background-image: url(images/social-icons.png);">&nbsp;</a>
							</div>
						</div>
						<div class="team__item">
							<div class="team__pic" style="background-image: url(images/team-item-2.png);"></div>
							<h2 class="team__name">Timothy Reed</h2>
							<span class="team__function">Co-Founder, Developer</span>
							<p class="team__descr">Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Maecenas sed diam eget risus varius blandit sit amet non magna. Nullam quis risus eget urna mollis ornare vel eu leo.</p>
							<div class="team__contacts social">
								<a href="#" class="social__link social__link_fb" style="background-image: url(images/social-icons.png);">&nbsp;</a><a href="#" class="social__link social__link_tw" style="background-image: url(images/social-icons.png);">&nbsp;</a><a href="#" class="social__link social__link_ln" style="background-image: url(images/social-icons.png);">&nbsp;</a>
							</div>
						</div>
						<div class="team__item">
							<div class="team__pic" style="background-image: url(images/team-item-3.png);"></div>
							<h2 class="team__name">Victoria Valdez</h2>
							<span class="team__function">UI Designer</span>
							<p class="team__descr">Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Maecenas sed diam eget risus varius blandit sit amet non magna. Nullam quis risus eget urna mollis ornare vel eu leo.</p>
							<div class="team__contacts social">
								<a href="#" class="social__link social__link_fb" style="background-image: url(images/social-icons.png);">&nbsp;</a><a href="#" class="social__link social__link_tw" style="background-image: url(images/social-icons.png);">&nbsp;</a><a href="#" class="social__link social__link_ln" style="background-image: url(images/social-icons.png);">&nbsp;</a>
							</div>
						</div>
						<div class="team__item">
							<div class="team__pic" style="background-image: url(images/team-item-4.png);"></div>
							<h2 class="team__name">Beverly Little</h2>
							<span class="team__function">Data Scientist</span>
							<p class="team__descr">Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Maecenas sed diam eget risus varius blandit sit amet non magna. Nullam quis risus eget urna mollis ornare vel eu leo.</p>
							<div class="team__contacts social">
								<a href="#" class="social__link social__link_fb" style="background-image: url(images/social-icons.png);">&nbsp;</a><a href="#" class="social__link social__link_tw" style="background-image: url(images/social-icons.png);">&nbsp;</a><a href="#" class="social__link social__link_ln" style="background-image: url(images/social-icons.png);">&nbsp;</a>
							</div>
						</div>
					</div>
				</div>
			</section>
			<!-- /team -->

			<!-- clients -->
			<section class="clients">
				<a id="clients"></a>
				<div class="clients__pic" style="background-image: url(images/clients.png);"></div>
				<div class="clients__descr">
					<div class="clients__inner">
						<blockquote class="clients__feedback">“Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Donec sed odio dui. Aenean eu leo quam...”</blockquote>
						<span class="clients__name">Susan Sims, Interaction Designer at XYZ</span>
						<div class="clients__links">
							<a href="#" class="clients__links-item">&nbsp;</a>
							<a href="#" class="clients__links-item clients__links-item_active">&nbsp;</a>
							<a href="#" class="clients__links-item">&nbsp;</a>
							<a href="#" class="clients__links-item">&nbsp;</a>
							<a href="#" class="clients__links-item">&nbsp;</a>
						</div>
					</div>	
				</div>
			</section>
			<!-- /clients -->

			<!-- download -->
			<section class="download">
				<a id="download"></a>
				<div class="container">
					<h2 class="download__title">Are You Ready to Start? Download Now For Free!</h2>
					<p class="download__descr">Fusce dapibus, tellus ac cursus commodo</p>
					<button class="download__btn button">Download here</button>
				</div>
			</section>
			<!-- /download -->

			<!-- footer -->
			<footer class="footer">
				<div class="container">
					<div class="footer__items">
						<div class="footer__location">
							<h2 class="footer__title">Location</h2>
							<address class="footer__descr">3481 Melrose Place<br>Beverly Hills, CA 90210</address>
						</div>
						<div class="footer__share">
							<h2 class="footer__title">Share with love</h2>
							<div class="footer__social social">
								<a href="#" class="social__link social__link_foo social__link_foo_fb" style="background-image: url(images/social-icons.png);">&nbsp;</a><a href="#" class="social__link social__link_foo social__link_foo_tw" style="background-image: url(images/social-icons.png);">&nbsp;</a><a href="#" class="social__link social__link_foo social__link_foo_ln" style="background-image: url(images/social-icons.png);">&nbsp;</a>
							</div>
						</div>
						<div class="footer__about">
							<h2 class="footer__title">About active box</h2>
							<p class="footer__descr">Cras justo odio, dapibus ac facilisis in, egestas eget quam. Donec ullamcorper nulla non metus auctor fringilla.</p>
						</div>
					</div>
				</div>
				<div class="footer__copyright copyright">
					<small class="copyright__info">Copyright &copy; 2015 ActiveBox. All Rights Reserved</small>
					<small class="copyright__info">Made with <span class="copyright__heart">Y</span><a href="#" class="copyright__author">by Kamal Chaneman</a></small>
				</div>
			</footer>
			<!-- /footer -->

		</div>
	</body>
</html>