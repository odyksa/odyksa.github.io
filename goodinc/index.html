<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8">
		<title>Good Inc.</title>
		<link rel="stylesheet" href="styles/main.css"> <!--load main styles-->
	</head>
	<body>
		<div id="wrapper">
			<div id="header">
				<div class="container clearfix">
					<div id="logo">
						<a href="#">g</a>
						<span>Good Inc.</span>
					</div>
				</div>
			</div>	
			<div id="menu">
				<div class="container clearfix">
					<ul id="nav" class="clearfix">
						<li><span><a href="#">Portfolio</a></span></li>
						<li><span><a href="#">Services</a></span></li>
						<li><span><a href="#">Staff</a></span></li>
						<li><span><a href="#">Articles</a></span></li>
						<li><span><a href="#">Contact</a></span></li>
					</ul>
				</div>
			</div>
			<div id="top-panel">
				<div class="container clearfix">
					<div id="search-panel">
						<form action="#">
							<input type="text" placeholder="Search for somthing here">
						</form>
					</div>
					<div id="social-panel">
						<a href="#" class="twr">Twitter</a>
						<a href="#" class="fbk">Facebook</a>
						<a href="#" class="rss">RSS</a>
						<a href="#" class="mail">Mail</a>
					</div>
				</div>
			</div>
			<div id="feature">
				<div class="container clearfix">
					<div class="feature-pic">
						<img src="images/feature-img.jpg" alt="feature-pic">
						<span>Good Inc. image caption to go here</span>
					</div>
					<div class="feature-txt">
						<h4><a href="#">Housework can't kill you, but why take a chance?</a></h4>
						<p>Quisque accumsan eleifend dolor, nec bibendum nibh ultricies at. Nunc convallis risus sit amet augue bibendum non lobortis eros interdum. Maecenas quis risus felis, at volutpat elit...</p>
						<span>14th April 2013</span>
					</div>
				</div>
			</div>
			<div id="articles">
				<div class="container clearfix">
					<div class="article-item">
						<div class="article-plus">
							<img src="images/add-triangle.png" alt="">
							<a href="#">+</a>
						</div>
						<div class="article-pic">
							<img class="article-img" src="images/articles-pic-1.jpg" alt="">
						</div>
						<div class="article-txt">
							<h2><a href="#" class="article-title-link">Taste & Sounds from Thailand</a></h2>
							<p>We've made some small changes to BlazRobar.com...</p>
						</div>
					</div>
					<div class="article-item">
						<div class="article-plus">
							<img src="images/add-triangle.png" alt="">
							<a href="#">+</a>
						</div>
						<div class="article-pic">
							<img class="article-img" src="images/articles-pic-2.jpg" alt="">
						</div>
						<div class="article-txt">
							<h2><a href="#" class="article-title-link">Taste & Sounds from Thailand</a></h2>
							<p>We've made some small changes to BlazRobar.com...</p>
						</div>
					</div>
					<div class="article-item">
						<div class="article-plus">
							<img src="images/add-triangle.png" alt="">
							<a href="#">+</a>
						</div>
						<div class="article-pic">
							<img class="article-img" src="images/articles-pic-3.jpg" alt="">
						</div>
						<div class="article-txt">
							<h2><a href="#" class="article-title-link">Taste & Sounds from Thailand</a></h2>
							<p>We've made some small changes to BlazRobar.com...</p>
						</div>
					</div>
				</div>
			</div>
			<div id="main">
				<div class="container clearfix">
					<div class="sidebar">
						<div class="article-item">
							<div class="article-plus">
								<img src="images/add-triangle.png" alt="">
								<a href="#">+</a>
							</div>
							<div class="article-pic">
								<img class="article-img" src="images/articles-pic-4.jpg" alt="">
							</div>
							<div class="article-txt">
								<h2><a href="#" class="article-title-link">Taste & Sounds from Thailand</a></h2>
								<p>Quisque accumsan eleifend dolor, nec bibendum nibh ultricies at. Nunc convallis risus sit amet augue bibendum non lobortis eros interdum. Maecenas quis risus felis, at volutpat elit. Proin vel nisl in orci venenatis suscipit eget sed lectus.</p>
							</div>
						</div>
						<div class="article-short">
							<h3><a href="#">Housework can't kill you, but why take a chance?</a></h3>
							<p>Quisque accumsan eleifend dolor, nec bibendum nibh ultricies at. Nunc convallis risus sit amet augue bibendum non lobortis eros interdum..</p>
							<span class="date">13th January 2013</span>
						</div>
						<div class="divider"></div>
						<div class="article-short">
							<h3><a href="#">My fake plants died because I did not pretend to water them</a></h3>
							<p>Quisque accumsan eleifend dolor, nec bibendum nibh ultricies at. Nunc convallis risus sit amet augue bibendum non lobortis eros interdum..</p>
							<span class="date">10th January 2013</span>
						</div>					
						<div class="additional-info">
							<ul>
								<li>
									<a href="#">About Us<span>(Blaz Robar)</span></a>
								</li>
								<li>
									<a href="#">How to<span>(Jamie Oliver)</span></a>
								</li>
								<li>
									<a href="#">Write for us<span>(Bill Muarry)</span></a>
								</li>
								<li>
									<a href="#">Video Testimonails<span>(John Smith)</span></a>
								</li>
							</ul>
						</div>
					</div>
					<div class="main-block">
						<div class="news-item">
							<h2><a href="#">Good Inc. Good Choice.</a></h2>
							<div class="info-panel">
								<span>14th January 2013</span>	
								<span class="divider"></span>
								<span>32 Comments</span>
								<span class="divider"></span>
								<span class="rating"></span>
								<span>124 Votes</span>
							</div>
							<p>I believe that if life gives you lemons, you should make lemonade... And try to find somebody whose life has given them vodka, and have a party.</p>
						</div>
						<div class="news-item orange-theme">
							<h2><a href="#" class="orange-theme">Behind every great man is a woman rolling</a></h2>
							<div class="info-panel">
								<span class="orange-theme">13th January 2013</span>	
								<span class="divider orange-theme"></span>
								<span class="orange-theme">312 Comments</span>
								<span class="divider orange-theme"></span>
								<span class="rating orange-theme"></span>
								<span class="orange-theme">1254 Votes</span>
							</div>
							<p class="orange-theme">I believe that if life gives you lemons, you should make lemonade... And try to find somebody whose life has given them vodka, and have a party.</p>
						</div>
						<div class="news-item short">
							<h2><a href="#">A day without sunshine is like, you know, night.</a></h2>
							<div class="info-panel">
								<span>14th January 2013</span>	
								<span class="divider"></span>
								<span>12 Comments</span>
							</div>
						</div>
						<div class="news-item short small-font">
							<h2><a href="#">Get your facts first, then you can distort them as you please.</a></h2>
							<div class="info-panel">
								<span>18th Fabruary 2013</span>	
								<span class="divider"></span>
								<span>24 Comments</span>
							</div>
						</div>
						<div class="news-item short small-font">
							<h2><a href="#">Quisque accumsan eleifend dolor, nec bibendum nibh ultricies at.</a></h2>
							<div class="info-panel">
								<span>17th March 2013</span>	
								<span class="divider"></span>
								<span>13 Comments</span>
							</div>
						</div>
						<div class="news-item short small-font">
							<h2><a href="#">Housework can't kill you, but why take a chance?</a></h2>
							<div class="info-panel">
								<span>19th January 2013</span>	
								<span class="divider"></span>
								<span>77 Comments</span>
							</div>
						</div>
						<div class="read-more">
							<span>Read more</span>
							<a href="#">+</a>
						</div>
						<div class="block-item">
							<div class="text-block-item">
								<h2><a href="#">Nice text boxage here</a></h2>
								<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec eu turpis purus. Pellentesque ullamcorper mauris vitae sapien bibendum tincidunt.</p>
								<p>Morbi malesuada mauris sed nibh placerat sit amet posuere libero egestas. Vestibulum consectetur iaculis est, at dapibus purus malesuada vitae. Mauris sollicitudin aliquam scelerisque. Vestibulum ante <span>ipsum primis in faucibus orci</span> luctus et ultrices posuere cubilia Curae; Donec et mi sem. Duis tristique elementum tristique. Sed ac magna quis erat sagittis suscipit. Phasellus faucibus rhoncus massa, non laoreet lorem fringilla non.</p>
							</div>
							<div class="footer-block-item clearfix">
								<span class="block-date">19th April</span>
								<span class="block-like">23</span>							
							</div>
						</div>
						<div class="article-item">
							<div class="article-pic">
								<img class="article-img" src="images/main-block-img.jpg" alt="">
							</div>
							<div class="article-txt">
								<h2><a href="#" class="article-title-link">I like long walks, especially when they are taken by people who really annoy me.</a></h2>
								<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec eu turpis purus. Pellentesque ullamcorper mauris vitae sapien bibendum tincidunt.</p>
								<p>Morbi malesuada mauris sed nibh placerat sit amet posuere libero egestas. Vestibulum consectetur iaculis est, at dapibus purus malesuada vitae. Mauris smassa, non laoreet lorem fringilla non.</p>
							</div>
						</div>
						<div class="article-item">
							<div class="article-txt">
								<h2><a href="#" class="article-title-link">A slightly different text box</a></h2>
								<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec eu turpis purus. Pellentesque ullamcorper mauris vitae sapien bibendum tincidunt. Check leo dignissim.</p>
								<p>Morbi malesuada mauris sed nibh placerat sit amet posuere libero egestas. Vestibulum consectetur iaculis est, at dapibus purus malesuada vitae. Mauris sollicitudin aliquam scelerisque. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Donec et mi sem. Duis tristique elementum tristique. Sed ac magna quis erat sagittis suscipit. Phasellus faucibus rhoncus massa, non laoreet lorem fringilla non.</p>
								<p>Mauris sollicitudin aliquam scelerisque. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Donec et mi sem. Duis tristique elementum tristique. Sed ac magna quis erat sagittis suscipit. Phasellus faucibus rhoncus.</p>
							</div>
						</div>
					</div>
				</div>
			</div>
			<div id="footer">
				<div class="container">
					<div class="clearfix">
						<ul>
							<li><a href="#">About us</a></li>
							<li><a href="#">Footer Links here</a></li>
							<li><a href="#"> Privacy</a></li>
							<li><a href="#">Page Links also</a></li>
						</ul>
					</div>
					<p>As cross as a hoon my built like a op shop. We're going bored sh*tless heaps she'll be right budgie smugglers. Grab us a durry mate she'll be right dunny rat. She'll be right larrikin with skite. As stands out like kero it'll be truckie.</p>
				</div>
			</div>
		</div>
	</body>
</html>