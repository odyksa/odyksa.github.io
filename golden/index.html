<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Golden Site</title>
	<link rel="stylesheet" href="styles/main.css"> <!--main styles-->
</head>
<body>
	<div id="wrapper">
		<div id="header">
			<div class="container clearfix">
				<a href="/" id="logo"><img src="images/logo.png" alt="logo" width="79" height="28"></a>
				<ul id="nav">
					<li><a href="#">Home</a></li>
					<li><a href="#">Services</a></li>
					<li><a href="#">Portfolio</a></li>
					<li><a href="#">About</a></li>
					<li><a href="#">Contact</a></li>
				</ul>
			</div>		

			<div class="container">
				<h1>Welcome to our studio!</h1>
				<p>It's nice to meet you</p>
				<input type="button" class="yellow-btn" value="Tell me more" />
			</div>	
		</div>
	
		<div id="services">
			<div class="container clearfix">
				<h2 class="block-title">Services</h2>
				<p class="block-sub">Proin iaculis purus consequat sem cure</p>
				<div class="item">
					<img src="images/services-icon-1.png" width="78" height="77" alt="">
					<h3 class="item-title">E-Commerce</h3>
					<p class="item-sub">Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit</p>
				</div>

				<div class="item">
				<img src="images/services-icon-2.png" width="78" height="77" alt="">
					<h3 class="item-title">Responsive Web</h3>
					<p class="item-sub">Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit</p>
				</div>

				<div class="item">
				<img src="images/services-icon-3.png" width="78" height="77" alt="">
					<h3 class="item-title">Web Security</h3>
					<p class="item-sub">Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit</p>
				</div>
			</div>
		</div>

		<div id="portfolio">
			<div class="container clearfix">
				<h2 class="block-title">Our portfolio</h2>
				<p class="block-sub">Proin iaculis purus consequat sem cure</p>
				<div class="item">
					<img src="images/portfolio-item-1.jpg" alt="" width="289" height="300">
					<h3 class="item-title">Ebony &amp; Ivory</h3>
					<p class="item-sub">Branding</p>
				</div>

				<div class="item">
					<img src="images/portfolio-item-2.jpg" alt="" width="289" height="300">
					<h3 class="item-title">Smart stationary</h3>
					<p class="item-sub">Print design</p>
				</div>

				<div class="item">
					<img src="images/portfolio-item-3.jpg" alt="" width="289" height="300">
					<h3 class="item-title">Clever poster</h3>
					<p class="item-sub">Print design</p>
				</div>

				<div class="item">
					<img src="images/portfolio-item-4.jpg" alt="" width="289" height="300">
					<h3 class="item-title">Vinyl record</h3>
					<p class="item-sub">Product mock-up</p>
				</div>

				<div class="item">
					<img src="images/portfolio-item-5.jpg" alt="" width="289" height="300">
					<h3 class="item-title">Treehouse template</h3>
					<p class="item-sub">Web design</p>
				</div>

				<div class="item">
					<img src="images/portfolio-item-6.jpg" alt="" width="289" height="300">
					<h3 class="item-title">Burned logo</h3>
					<p class="item-sub">Branding</p>
				</div>
			</div>
		</div>

		<div id="about">
			<div class="container clearfix">
				<h2 class="block-title">About us</h2>
				<p class="block-sub">Proin iaculis purus consequat sem cure</p>
				<div class="item clearfix">
					<div class="about-text side-left">
						<h3><span>July 2010</span><br />Our Hhmble beginnings</h3>
						<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni 
						ssim. Donec porttitora entum suscipit. Proin iaculis purus consequat sem cure 
						digni ssim.</p>
					</div>
					<div class="about-pic side-left">
						<img src="images/about-pic-1.png" alt="">
					</div>
				</div>

				<div class="item clearfix">
					<div class="about-text side-right">
						<h3><span>January 2011</span><br />Facing startup battles</h3>
						<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni 
						ssim. Donec porttitora entum suscipit.</p>
					</div>
					<div class="about-pic side-right">
						<img src="images/about-pic-2.png" alt="">
					</div>
				</div>

				<div class="item clearfix">
					<div class="about-text side-left">
						<h3><span>December 2012</span><br />Enter the dark days</h3>
						<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni 
						ssim. Donec porttitora entum suscipit.</p>
					</div>
					<div class="about-pic side-left">
						<img src="images/about-pic-3.png" alt="">
					</div>
				</div>

				<div class="item clearfix">
					<div class="about-text side-right">
						<h3><span>Fabruary 2014</span><br />Facing startup battles</h3>
						<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni 
						ssim. Donec porttitora entum suscipit. Proin iaculis purus consequat sem cure 
						digni ssim.</p>
					</div>
					<div class="about-pic side-right">
						<img src="images/about-pic-4.png" alt="">
					</div>
				</div>

				<div class="item">
					<a href="#" class="about-yellow">Our<br />story<br />continues<br />...</a>
				</div>
			</div>
		</div>

		<div id="team">
			<div class="container">
				<h2 class="block-title">Our amazing team</h2>
				<p class="block-sub">Proin iaculis purus consequat sem cure</p>
				<div class="items-wrap clearfix">
					<div class="item">
						<div class="circle"></div>
						<h3 class="item-title">Kimberly Thompson</h3>
						<p class="item-sub">Project manager</p>
						<div class="social">
							<a href="#" class="twitter"></a>
							<a href="#" class="facebook"></a>
							<a href="#" class="pinterest"></a>
						</div>
					</div>

					<div class="item">
						<div class="circle"></div>
						<h3 class="item-title">Rico Massimo</h3>
						<p class="item-sub">Coder</p>
						<div class="social">
							<a href="#" class="twitter"></a>
							<a href="#" class="facebook"></a>
							<a href="#" class="google"></a>
						</div>
					</div>

					<div class="item">
						<div class="circle"></div>
						<h3 class="item-title">Uku Mason</h3>
						<p class="item-sub">Graphic designer</p>
						<div class="social">
							<a href="#" class="facebook"></a>
							<a href="#" class="pinterest"></a>
						</div>
					</div>
				</div>
				<p class="about-footer">Proin iaculis purus consequat sem cure  digni ssim donec porttitora entum suscipit<br />aenean rhoncus posuere odio in tincidunt proin iaculis.</p>
			</div>
		</div>

		<div id="brands">
			<div class="container">
				<a href="#"><img src="images/envato.jpg" width="161" height="30" alt=""></a>
				<a href="#"><img src="images/word-press.jpg" width="186" height="42" alt=""></a>
				<a href="#"><img src="images/tuts.jpg" width="131" height="41" alt=""></a>
				<a href="#"><img src="images/microlancer.jpg" width="167" height="42" alt=""></a>
			</div>
		</div>

		<div id="contact">
			<div class="container">
				<h2 class="block-title">Contact us</h2>
				<p class="block-sub">Proin iaculis purus consequat sem cure</p>
				<form action="#" method="#">
					<div class="clearfix">						
						<div class="form-left">
							<input type="text" placeholder="Your name*" name="name">
							<input type="text" placeholder="Your e-mail*" name="email">
							<input type="text" placeholder="Subject" name="subject">
						</div>

						<textarea class="form-right" placeholder="Your message*" name="comments"></textarea>
					</div>
					<input type="submit" value="Send message" class="yellow-btn" />
				</form>
			</div>
		</div>

		<div id="footer">
			<div class="container">
				<p class="copyright">&copy; Copyright 2014 <a href="FreebiesXpress.com">FreebiesXpress.com</a></p>
				<div class="social">
					<a href="#" class="twitter"></a>
					<a href="#" class="facebook"></a>
					<a href="#" class="pinterest"></a>
					<a href="#" class="google"></a>			
				</div>
			</div>
		</div>
	</div>
</body>
</html>